src/functionalTest/resources/request/casedata/ccd-callback-casedata-citizen-submit-conditional-order.json

```diff
@@ -1,12 +1,13 @@
 {
   "divorceOrDissolution": "divorce",
   "applicationType": "jointApplication",
-  "coDateSubmitted": null,
+  "coApplicant1SubmittedDate": "2022-01-18T12:12:12.123",
```

Comment:
was the field name changed from coDateSubmitted -> coApplicant1SubmittedDate ? if yes it will need migration in RetiredFields

Reply:
yes, but that was done in a separate PR and it was added to the retired fields